ckanext/datatablesview/templates/datatables/datatables_form.html

```diff
@@ -6,27 +6,31 @@
   label=_('Responsive display'),
   value='True',
   checked=data.responsive,
+  attrs={"title":"Optimize table layout for smaller screen sizes"},
```

Comment:
What is the functionality of attrs? Is it only backend comments?looks like not appearing anywhere at front end

Reply:
If you hover over the checkbox the title will appear informing users of what it's for.

Reply:
Thanks. I hovered over the text following instead of the box.